app/src/main/java/io/intrepid/skeleton/base/PresenterConfiguration.java

```diff
@@ -6,11 +6,14 @@
 import io.intrepid.skeleton.settings.UserSettings;
 import rx.Scheduler;
 
+/**
+ * Wrapper class for common dependencies that all presenters are expected to have
+ */
 public class PresenterConfiguration {
     @NonNull
-    protected final Scheduler ioScheduler;
+    private final Scheduler ioScheduler;
     @NonNull
-    protected final Scheduler uiScheduler;
+    private final Scheduler uiScheduler;
     @NonNull
     private final UserSettings userSettings;
     @NonNull
```

Comment:
Do we want to add `CrashReporter` and `AnalyticsTracker` as injected dependencies to the presenter config?